virtual-desktop/src/app/plugin-manager/plugin-factory/iframe/iframe-plugin.component.ts

```diff
@@ -72,8 +72,10 @@ export class IFramePluginComponent {
             pluginDef: {},
             launchMetadata: this.launchMetadata
           },
-          instanceId: this.instanceId
-        }, '*')
+          instanceId: this.instanceId,
+          error: 'Unable to parse plugin definition'
+        }, '*');
+        console.warn('Unable to parse plugin defintion');
```

Comment:
This needs to be a logger owned by the desktop.
Give this a try:
import { BaseLogger } from 'virtual-desktop-logger';
then just set this.logger to BaseLogger.